ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -0,0 +1,188 @@
+const LOOKBACK_PERIOD = 330;
+
+const renewalStyle = {
+  h1: {
+    fontSize: '0.83rem',
+    fontWeight: 600,
+    color: colors.primary
+  },
+  h2: {
+    fontSize: '0.75rem',
+    fontWeight: 600
+  },
+  h3: {
+    fontSize: '0.675rem',
+    fontWeight: 600
+  },
+  card: {
+    backgroundColor: colors.white,
+    border: `1px solid ${colorWithWhiteness(colors.dark, 0.8)}`,
+    borderRadius: '0.2rem',
+    display: 'flex',
+    fontSize: '0.58rem',
+    fontWeight: 500,
+    height: 345,
+    lineHeight: '22px',
+    margin: 0,
+    padding: '0.5rem',
+    width: 560
+  }
+};
+
+const BackArrow = withCircleBackground(() => <Arrow style={{height: 21, width: 18}}/>);
+
+const RenewalCard = withStyle(renewalStyle.card)(
+  ({step, title, TitleComponent, lastCompletion, nextReview, children, style}) => {
+    const daysRemaining = daysFromNow(nextReview);
+    return <FlexColumn style={style}>
+      <div style={renewalStyle.h3}>STEP {step}</div>
+      <div style={renewalStyle.h3}><TitleComponent/></div>
+      <div style={{ color: colors.primary, margin: '0.5rem 0', display: 'grid', gridTemplateColumns: '6rem 1fr'}}>
+        <div>Last Updated On:</div>
+        <div>Next Review:</div>
+        <div>{`${displayDateWithoutHours(lastCompletion)}`}</div>
+        <div>{`${displayDateWithoutHours(nextReview)} (${daysRemaining > 0 ? daysRemaining + ' days' : 'expired'})`}</div>
+      </div>
+      {children}
+    </FlexColumn>;
+  }
+);
+
+const CompletedButton = ({buttonText, wasBypassed, style}:
+  {buttonText: string, wasBypassed: boolean, style?: React.CSSProperties}) => <Button disabled={true}
+                      data-test-id='completed-button'
+                      style={{
+                        height: '1.6rem',
+                        marginTop: 'auto',
+                        backgroundColor: colors.success,
+                        width: 'max-content',
+                        cursor: 'default',
+                        ...style
+                      }}>
+    <ClrIcon shape='check' style={{marginRight: '0.3rem'}}/>{wasBypassed ? 'Bypassed' : buttonText}
+  </Button>;
+
+const getExpirationTimeForModule = fp.curry((modules, moduleName) => {
+  return fp.flow(fp.find({moduleName: moduleName}), fp.get('expirationEpochMillis'))(modules);
+});
+
+const isComplete = (nextReview: number): boolean => daysFromNow(nextReview) > LOOKBACK_PERIOD;
+
+export const AccessRenewalPage = fp.flow(
+  withRouteData,
+  withProfileErrorModal
+)(() => {
+  const {profile: {
+    complianceTrainingCompletionTime,
+    dataUseAgreementCompletionTime,
+    publicationsLastConfirmedTime,
+    profileLastConfirmedTime,
+    dataUseAgreementBypassTime,
+    complianceTrainingBypassTime,
+    renewableAccessModules: {modules}},
+  } = useStore(profileStore);
+  const [publications, setPublications] = useState<boolean>(null);
+  const noReportId = useId();
+  const reportId = useId();
+  const getExpirationTimeFor = getExpirationTimeForModule(modules);
```

Comment:
We don't have much currying in the codebase right now. I'm not dead set against it if it improves readability, but in this case it feels like it might be simpler to just inline the callback here and reference `modules` via closure, since this is only used in one location.